coral/src/app/features/topics/request/components/AdvancedConfiguration.tsx

```diff
@@ -67,7 +67,7 @@ function AdvancedConfiguration({ name }: Props) {
   }
 
   return (
-    <Box component={Flexbox} gap={"l1"} direction={"column"}>
+    <Box component={Box.Flex} gap={"l1"} direction={"column"}>
```

Comment:
I don't know how `<Box component={Flexbox} .../>` sneaked in from our side 😅  

Could you update your change to use `<Box.Flex>` to be consistent?

Reply:
I have done that. Could you take a look and tell me if there's something I missed